test/fairtools/_TestFairTools.cxx

```diff
@@ -71,323 +62,321 @@ template <class T> class _TestFairLoggerBase : public T
+        fLogger->Info(MESSAGE_ORIGIN, OutputString.c_str());
+        fLogger->Debug(MESSAGE_ORIGIN, OutputString.c_str());
+        fLogger->Debug1(MESSAGE_ORIGIN, OutputString.c_str());
+        fLogger->Debug2(MESSAGE_ORIGIN, OutputString.c_str());
+        fLogger->Debug3(MESSAGE_ORIGIN, OutputString.c_str());
+        fLogger->Debug4(MESSAGE_ORIGIN, OutputString.c_str());
     }
 
-    std::vector<std::string> CreateExpectedOutputNoArguments(const std::string& loglevel, const std::string& outString, const std::string& filename = "") {
-
-      //Put blanks to the string to have all strings the same length
-      int size = loglevel.size();
-      int maxsize = 7; // length of string "WARNING"
-      for (int i=size; i<maxsize; i++) {
-        loglevel += " ";
-      }
-
-      std::vector<std::string> levelNames(LogString, LogString+9);
-      std::vector<std::string>::iterator position =
-        std::find(levelNames.begin(), levelNames.end(), loglevel);
-      levelNames.erase(position+1, levelNames.end());
-      levelNames.erase(levelNames.begin()); // don't use FATAL loglevel in this test
-
-      std::vector<std::string> retVal;
-      std::string fullOutputString;
-      if ( filename.compare("") != 0) {
-        fullOutputString = "[INFO   ] Openning log file: " + filename;
-        retVal.push_back(fullOutputString);
-        retVal.push_back(" ");
-      }
-
-      std::vector<std::string>::iterator iter;
-      for ( iter = levelNames.begin();
-            iter < levelNames.end();
-            iter++) {
-        fullOutputString = "[" + *iter + "] " + outString + " ";
-        retVal.push_back(fullOutputString);
-      }
-
-      return retVal;
+    std::vector<std::string> CreateExpectedOutputNoArguments(const std::string& loglevel,
+                                                             const std::string& outString,
+                                                             const std::string& filename = "")
+    {
+
+        // Put blanks to the string to have all strings the same length
+        int size = loglevel.size();
+        int maxsize = 7;   // length of string "WARNING"
+        for (int i = size; i < maxsize; i++) {
+            loglevel += " ";
+        }
+
+        std::vector<std::string> levelNames(LogString, LogString + 9);
+        std::vector<std::string>::iterator position = std::find(levelNames.begin(), levelNames.end(), loglevel);
+        levelNames.erase(position + 1, levelNames.end());
+        levelNames.erase(levelNames.begin());   // don't use FATAL loglevel in this test
+
+        std::vector<std::string> retVal;
+        std::string fullOutputString;
+        if (filename.compare("") != 0) {
+            fullOutputString = "[INFO   ] Openning log file: " + filename;
+            retVal.push_back(fullOutputString);
+            retVal.push_back(" ");
+        }
+
+        std::vector<std::string>::iterator iter;
+        for (iter = levelNames.begin(); iter < levelNames.end(); iter++) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/manual/FairRootGroup/FairRoot/pullRequest?prid=5323596)